server/api/themoviedb/index.ts

```diff
@@ -1064,6 +1064,9 @@ class TheMovieDb extends ExternalAPI {
 
       return data;
     } catch (e) {
+      if (e.response?.status === 404) {
+        throw e;
+      }
```

Comment:
If the error is expected, you could instead return null? IMO it's a cleaner way to say that this is expected to have no data than throwing another type of error